pom.xml

```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509</version>
+    <version>1.609.1</version>
```

Comment:
If i right understand this plugin provides branch features for job and that's what i'm long waiting for git plugin. So if it is, then i will be glad to see not updated core, git and many other tools (that not yet ..ed by workflow) works on 1.565. I think we can also start operate with statistics and verify how many < 1.609.1 are exists before enforcing updates without reason.